orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -3,6 +3,8 @@
 
 #include "ort_aten.h"
 #include "ort_tensor.h"
+#include "ort_backends.h"
+#include <core/providers/cpu/cpu_execution_provider.h>
```

Comment:
why we need those headers?

Reply:
I think we need those headers to create OrtMemoryInfo

`  OrtMemoryInfo *mem_info;
  Ort::ThrowOnError(Ort::GetApi().CreateCpuMemoryInfo(OrtArenaAllocator, OrtMemTypeDefault, &mem_info));`

Reply:
Fixed